### PR TITLE
Add DD_URL for datadog-operator chart and update values

### PR DESCRIPTION
**Pull request by paktek123**
#### What this PR does / why we need it:

This PR adds the ability to configure the datadog api endpoint for the datadog-operator Helm Chart.

#### Which issue this PR fixes
fixes https://github.com/DataDog/helm-charts/issues/260

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] Documentation has been updated with helm-docs (run: `.github/helm-docs.sh`)
- [x] Chart Version bumped
- [x] `CHANGELOG.md` has beed updated
- [x] Variables are documented in the `README.md`

### CODE REVIEW

**Comment by paktek123** [comment]
how do I update the helm docs? is that an action for me?

**Comment by paktek123** [comment]
ok did it

File: charts/datadog-operator/values.yaml [hunk 1]
```diff
@@ -15,6 +15,9 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# DataDog API endpoint, example: https://api.datadoghq.eu
+dataDogUrl: #<DATADOG_API_ENDPOINT>
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:19]: Could you used the same variable name (dd_url) that in the `datadog/datadog` chart? https://github.com/DataDog/helm-charts/blob/master/charts/datadog/values.yaml#L67

**Review by clamoriniere — COMMENTED** [review]
Hi @paktek123 

Thank you for opening this PR. you change looks good. 
Just a small comment about the name of the new field.

File: charts/datadog-operator/values.yaml (hunk 1 above)
Comment (paktek123) [reply charts/datadog-operator/values.yaml]: Done!

File: charts/datadog-operator/values.yaml [hunk 2]
```diff
@@ -15,6 +15,9 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# DataDog API endpoint, example: https://api.datadoghq.eu
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:18]: lets also copy the description of the variable from the `datadog/datadog` chart.

```suggestion
# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL  
## Overrides the site setting defined in "site".
```

File: charts/datadog-operator/README.md [hunk 3]
```diff
@@ -15,6 +15,7 @@
 | datadog-crds.crds.datadogMetrics | bool | `true` | Set to true to deploy the DatadogMetrics CRD |
 | datadog-crds.crds.datadogMonitors | bool | `true` | Set to true to deploy the DatadogMonitors CRD |
 | datadogMonitor.enabled | bool | `false` | Enables the Datadog Monitor controller |
+| dd_url | string | `nil` |  |
```
[the remark this answers is not in the document]
Comment (paktek123) [reply charts/datadog-operator/README.md]: ran it now

File: charts/datadog-operator/values.yaml [hunk 4]
```diff
@@ -15,6 +15,10 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL  
+## Overrides the site setting defined in "site".
+dd_url: #<DATADOG_API_ENDPOINT>
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:20]:
```suggestion
dd_url: # <DATADOG_API_ENDPOINT>
```

File: charts/datadog-operator/values.yaml [hunk 5]
```diff
@@ -15,6 +15,10 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:18]:
```suggestion
# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL .
```

File: charts/datadog-operator/values.yaml [hunk 6]
```diff
@@ -15,6 +15,10 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL  
+## Overrides the site setting defined in "site".
+dd_url: #<DATADOG_API_ENDPOINT>
+
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:21]:
```suggestion


```

File: charts/datadog-operator/values.yaml [hunk 7]
```diff
@@ -15,6 +15,11 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
 # appKey -- Your Datadog APP key
 appKey:  # <DATADOG_APP_KEY>
 
+# dd_url -- The host of the Datadog intake server to send Agent data to, only set this option if you need the Agent to send data to a custom URL
+## Overrides the site setting defined in "site".
+dd_url: # <DATADOG_API_ENDPOINT>
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:20]:
```suggestion
dd_url:  # <DATADOG_API_ENDPOINT>
```

File: charts/datadog-operator/values.yaml [hunk 8]
```diff
@@ -15,6 +15,11 @@ apiKeyExistingSecret:  # <DATADOG_API_KEY_SECRET>
...
+
+
```
Comment (clamoriniere) [inline charts/datadog-operator/values.yaml:22]:
```suggestion
```

**Review by clamoriniere — APPROVED** [review]

**Pull request merged**